src/main/java/br/com/vivo/model/RegistroCdr.java

```diff
@@ -0,0 +1,38 @@
+package br.com.vivo.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import br.com.vivo.enums.Formato;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "registro_cdr")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
```

Comment:
coloca essa anotação -> @EqualsAndHashCode(of = "{id}")

Reply:
Feito!